Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.h

```diff
@@ -495,6 +520,18 @@ namespace AssetProcessor
         typedef QHash<QString, FileEntry> FileExamineContainer;
         FileExamineContainer m_filesToExamine; // order does not actually matter in this (yet)
 
+        // Set of files which are metadata-enabled but don't have a metadata file.
+        // These files will be delayed for processing for a short time to wait for a metadata file to show up.
+        AZStd::unordered_map<AZ::IO::Path, QDateTime> m_delayProcessMetadataFiles;
```

Comment:
Do we have any concerns about size at edge cases here? Do we have a sense of memory footprint per tracked file? Imagine this set should always be small(ish)

Reply:
Shouldn't be any real concerns, there are lots of other maps APM is managing and this one will be cleared out every few seconds